fedimint-cli/Cargo.toml

```diff
@@ -31,9 +31,9 @@ bitcoin = { workspace = true }
 clap = { workspace = true }
 clap_complete = "4.5.40"
 fedimint-aead = { workspace = true }
-fedimint-api-client = { path = "../fedimint-api-client", version = "=0.6.0-alpha", default-features = false }
+fedimint-api-client = { path = "../fedimint-api-client", version = "=0.6.0-alpha", features = ["tor"] }
```

Comment:
Not needed. The `tor = ["fedimint-client/tor", "fedimint-api-client/tor"]` already does it.